src/components/structures/RoomView.tsx

```diff
@@ -914,6 +929,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
         const callState = call?.state;
         this.setState({
             callState,
+            isRoomEncrypted: await this.getIsRoomEncrypted(),
```

Comment:
This seems superfluous, what could have set `state.roomId` at this point in the flow?